pkg/logs/config/config.go

```diff
@@ -184,13 +184,28 @@ func buildTCPEndpoints(logsConfig *LogsConfigKeys) (*Endpoints, error) {
 		main.UseSSL = !logsConfig.devModeNoSSL()
 	}
 
+	var backup *Endpoint
+
+	if logsDDBackupURL, defined := logsConfig.logsDDURLBackup(); defined {
+		// Copy the main endpoint since all the settings are shared except the host + port
+		copiedEndpoint := main
+		host, port, err := parseAddress(logsDDBackupURL)
+		if err != nil {
+			return nil, fmt.Errorf("could not parse %s: %v", logsDDBackupURL, err)
+		}
+		copiedEndpoint.Host = host
+		copiedEndpoint.Port = port
+
+		backup = &copiedEndpoint
+	}
```

Comment:
additional endpoints currently make no guarantees about log retention. It's a best effort "fire and forget" and they won't block the pipeline when they fail. I think we should preserve that behavior for customers that rely on it existing and behaving that way since the backup sender will incur additonal overhead. WDYT?

Reply:
Yeah that's a tricky one and makes sense. It would be nice to treat all of these non-primary endpoints as "not pets" and use a single logic flow for them from a maintenance/resilience standpoint but it's a massive change that probably doesn't need to co-mingle with this PR nor be done at this exact time.

Reply:
I agree with @sgnn7 that this is a major change, but maybe as a followup?  I think that customers would look at this as "I have these destinations for logs, and I'm willing to accept lost log messages to this subset of destinations instead of buffering on the agent".

It might be nice to see if we could get that config interface "right" now, even if some of the possible configurations are not supported, so that we could add such support in a followup.